src/models/navigation.ts

```diff
@@ -50,12 +50,17 @@ export type NavigationItemData =
 
 export type DropdownItemData = Omit<NavigationDropdownItem, 'items'>;
 
-export interface NavigationLogoData {
+export interface NavigationLogoThemedData {
     icon: ImageProps;
     text?: string;
     url?: string;
 }
 
+export interface NavigationLogoData extends NavigationLogoThemedData {
```

Comment:
we use type util `ThemeSupporting` for cases like that
there are example in `src/models/constructor-items/blocks.ts`

Reply:
fixed, types union use for prevent breaking changes